playwright.config.ts

```diff
@@ -21,16 +21,27 @@ const config: PlaywrightTestConfig = {
   workers: process.env.CI ? 1 : undefined,
   testDir: 'tests',
   timeout: 120 * 1000,
+  reporter: process.env.CI ? 'blob' : 'dot',
```

Comment:
blob is used for sharding CI and then putting them together for a report. otherwise we can use `dot` or any other reporter we fancy